src/main/java/com/github/alexthe666/iceandfire/item/IafItemRegistry.java

```diff
@@ -306,7 +307,7 @@ public class IafItemRegistry {
     @GameRegistry.ObjectHolder(IceAndFire.MODID + ":myrmex_desert_sword")
     public static Item myrmex_desert_sword = new ItemModSword(myrmexChitin, "myrmex_desert_sword", "iceandfire.myrmex_desert_sword", 1);
     @GameRegistry.ObjectHolder(IceAndFire.MODID + ":myrmex_desert_sword_venom")
-    public static Item myrmex_desert_sword_venom = new ItemModSword(myrmexChitin, "myrmex_desert_sword_venom", "iceandfire.myrmex_desert_sword_venom", 2);
+    public static Item myrmex_desert_sword_venom = new ItemModSword(myrmexChitin, "myrmex_desert_sword_venom", "iceandfire.myrmex_desert_sword_venom", 5);
```

Comment:
As mentioned before- revert this change to make it easier to rebase onto main branch